### PR TITLE
[API] Dedicated API for Conv+Bias+Activ Fusion

### PR DESCRIPTION
**Pull request by JehandadKhan**

### CODE REVIEW

**Comment by JehandadKhan** [comment]
In draft pending addition of  more tests

File: test/gtest/api_convbiasactiv.cpp [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        std::fill(output.begin(), output.end(), 0.0f);
+        std::fill(ref_out.begin(), ref_out.end(), 0.0f);
+        std::fill(bias.begin(), bias.end(), 0.0f);
+        int bias_mode = 1; // zero disables bias
+        convHostForward(input, ref_out, weights, bias_mode, bias, conv_desc);
+        activationHostInfer(cba_config.activ_mode,
+                            double_zero,
+                            double_zero,
+                            double_zero,
+                            ref_out.data,
```
Comment (xinlipn) [inline test/gtest/api_convbiasactiv.cpp:107]: Should this be input.data instead?

Comment (xinlipn) [inline test/gtest/api_convbiasactiv.cpp:107]: Use non-zero alpha, beta, gamma value here?

Comment (junliume) [reply test/gtest/api_convbiasactiv.cpp:107]: @atamazov @DrizztDoUrden could you take a look while @JehandadKhan is on vacation?

**Review by junliume — APPROVED** [review]

**Pull request merged**